zerver/openapi/zulip.yaml

```diff
@@ -15484,20 +15484,20 @@ paths:
 
         Clients implementing Zulip's typing notifications protocol should work as follows:
 
-        - Send a request to this endpoint with `op = "start"` when a user starts typing a message,
-          and also every `TYPING_STARTED_WAIT_PERIOD = 10` seconds that the user continues to
+        - Send a request to this endpoint with `"op": "start"` when a user starts typing a message,
+          and also every `"TYPING_STARTED_WAIT_PERIOD": 10` seconds that the user continues to
```

Comment:
This one really is meant as an equals sign — `TYPING_STARTED_WAIT_PERIOD` isn't in the API, and doesn't appear anywhere as JSON like this. OTOH the equals sign probably shouldn't be in code font (and this should likely be reworded.)

What's meant here is "every `TYPING_STARTED_WAIT_PERIOD` seconds, and by the way `TYPING_STARTED_WAIT_PERIOD` equals 10." With an equals sign, it's saying that in the way one typically would in mathematical prose.

Similarly for `TYPING_STOPPED_WAIT_PERIOD` and `TYPING_STARTED_EXPIRY_PERIOD` below.

Probably the ideal fix here is that we should put those three timing constants into the API, with the server announcing them in /api/register-queue, like we recently did for presence; and then this discussion can refer to them by the names they have there. May not be worth investing that effort into this feature right now, though.

Pending that, perhaps something like this:
> […] every 10 seconds (`TYPING_STARTED_WAIT_PERIOD`) that […]

Reply:
Thanks for catching that! I've started a conversation [here on CZO](https://chat.zulip.org/#narrow/stream/378-api-design/topic/typing.20notifications.20constants/near/1570027.2E01).

And I updated these changes per your suggestion; see updated screenshot below. I think this version better highlights the actual times in seconds for these, instead of the constants.

<details>
<summary>Updates to Set typing status main description</summary>

![Screenshot from 2023-05-16 11-42-26](https://github.com/zulip/zulip/assets/63245456/1dde5ad4-ce95-4bd8-a32a-6597116c6611)
</details>